kong/plugins/ai-prompt-guard/schema.lua

```diff
@@ -8,19 +8,19 @@ return {
       type = "record",
       fields = {
           { allow_patterns = {
-              description = "Array of valid patterns, or valid questions from the 'user' role in chat.",
+              description = "Array of valid regex patterns, or valid questions from the 'user' role in chat.",
               type = "array",
-              default = {},
+              required = false,
               len_max = 10,
               elements = {
                 type = "string",
                 len_min = 1,
                 len_max = 500,
```

Comment:
He have regex checking on the schema already, perhaps this would work?
```suggestion
                len_max = 500,
                is_regex = true,
```